bigframes/session/executor.py

```diff
@@ -329,7 +329,7 @@ def export_gbq(
         if if_exists != "append" and has_timedelta_col:
             # Only update schema if this is not modifying an existing table, and the
             # new table contains timedelta columns.
-            assert query_job.destination is not None
+            assert query_job is not None and query_job.destination is not None
             table = self.bqclient.get_table(query_job.destination)
```

Comment:
I'm curious why we don't use the `destination` argument for this instead of fetching from the job.